application/modules/shop/assets/BatchEditPriceAsset.php

```diff
@@ -0,0 +1,12 @@
+<?php
+namespace app\modules\shop\assets;
+
+use yii\web\AssetBundle;
+
+class BatchEditPriceAsset extends AssetBundle
+{
+    public $sourcePath = '@app/modules/shop/assets';
```

Comment:
it's a bad idea to make functional-specific asset bundle inside module specific category